### PR TITLE
stm32cube: Remove all STM32Cube release notes

### PR DESCRIPTION
**Pull request by benothmn-st**
Remove all STM32Cube release notes since the links to release notes of all components are relative paths to Cube package. So, they are broken links.
In the README file of the each cube folder, there is a link to the GitHub containing the release notes and all the STM32Cube details.

### CODE REVIEW

**Review by erwango — APPROVED** [review]
Approved, but DNM until v4.0.0 is released

**Review by FRASTM — APPROVED** [review]

**Review by erwango — CHANGES_REQUESTED** [review]
Back on this.

Removing release notes is ok, but since these files are updated automatically by series update scripts, the scripts should be updated to in order to not update the release notes from now on (otherwise they'll be back at next update).

So please update scripts additionally (in same PR but another commit).

**Comment by benothmn-st** [comment]
@ABOSTM could you review this PR please?

File: scripts/serie_update.py [hunk 1]
```diff
@@ -640,10 +640,9 @@ def apply_zephyr_patch(self):
             os.remove(patch_path)
         shutil.copy(str(self.stm32cube_temp / self.module_patch), patch_path)
 
-        # Update README and CMakeList, copy release note
+        # Update README and CMakeList
         self.update_readme(self.version_update, self.update_commit)
         self.update_cmakelist()
-        self.copy_release_note()
```
Comment (erwango) [inline scripts/serie_update.py:646]: That's part of it, but let's remove the method as well to minimize dead code.

**Review by erwango — CHANGES_REQUESTED** [review]

**Review by erwango — APPROVED** [review]

**Pull request merged**